gqlauth/core/scalars.py

```
@@ -18,36 +19,39 @@ def serialize_excpected_error(errors):
-    }
-    """,
-    serialize=lambda value: serialize_excpected_error(value),
-    parse_value=lambda value: value,
-)
+if TYPE_CHECKING:
+    ExpectedErrorType = dict[Any, Any] | list[dict[Any, Any]]
+else:
+    ExpectedErrorType = strawberry.scalar(
+        typing.NewType("ExpectedError", dict),
+        description="""
+	     Errors messages and codes mapped to
```

Flake8 (7.3.0)

[error] indentation contains mixed spaces and tabs

(E101)